specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
-              "$ref": "#/definitions/AcrErrors"
-            }
-          }
-        }
-      }
-    },
-    "/{nextBlobUuidLink}": {
-      "get": {
-        "tags": [
-          "ContainerRegistryBlob"
-        ],
-        "parameters": [
-          {
-            "$ref": "#/parameters/NextLink"
-          }
-        ],
-        "description": "Retrieve status of upload identified by uuid. The primary purpose of this endpoint is to resolve the current status of a resumable upload.",
-        "operationId": "ContainerRegistryBlob_GetUploadStatus",
-        "x-ms-examples": {
-          "Get blob status": {
-            "$ref": "./examples/GetBlobStatus.json"
+          "example": {
+              "registry": "registry",
+              "imageName": "imageName",
+              "tags": [
+                  {
+                      "changeableAttributes": {
+                          "readEnabled": true,
+                          "listEnabled": true,
+                          "deleteEnabled": true,
+                          "writeEnabled": true
+                      },
+                      "name": "name",
+                      "digest": "digest",
+                      "createdTime": "createdTime",
+                      "signed": true,
+                      "lastUpdateTime": "lastUpdateTime"
+                  },
+                  {
+                      "changeableAttributes": {
+                          "readEnabled": true,
+                          "listEnabled": true,
+                          "deleteEnabled": true,
+                          "writeEnabled": true
+                      },
+                      "name": "name",
+                      "digest": "digest",
+                      "createdTime": "createdTime",
+                      "signed": true,
+                      "lastUpdateTime": "lastUpdateTime"
+                  }
+              ]
           }
-        },
-        "responses": {
-          "204": {
-            "description": "The upload is known and in progress. The last received offset is available in the Range header.",
-            "headers": {
-              "Range": {
-                "description": "Range indicating the current progress of the upload.",
-                "type": "string"
-              },
-              "Docker-Upload-UUID": {
-                "description": "Identifies the docker upload uuid for the current request.",
-                "type": "string"
-              }
-            }
+      },
+      "TagAttributes": {
+          "required": [
+              "registry",
+              "imageName",
+              "tag"
+          ],
+          "x-ms-client-name": "ArtifactTagProperties",
+          "description": "Tag attributes",
+          "properties": {
+              "registry": {
+                  "type": "string",
+                  "readOnly": true,
+                  "description": "Registry login server name.  This is likely to be similar to {registry-name}.azurecr.io",
```

Comment:
The domain can be different for different clouds though, can we include it in the samples instead of in the swagger?

Reply:
@shahzzzam fyi